src/doc/rustc/src/platform-support/win7-windows-gnu.md

```diff
@@ -0,0 +1,44 @@
+# \*-win7-windows-gnu
+
+**Tier: 3**
+
+Windows targets continuing support of Windows 7.
```

Comment:
Suggestion: can you enumerate the two targets here? As in, explicitly list out:

- `x86_64-win7-windows-gnu`
- `i686-win7-windows-gnu`

to make it easier to search for?

Reply:
Done.